condor_watch_q.py

```diff
@@ -483,34 +484,48 @@ def table_by(clusters, attribute, abbreviate_path_components):
                 totals[status] += row_data[status]
 
         row_data[key] = attribute_value
-
         rows.append(row_data)
 
     if key == EVENT_LOG:
         for row in rows:
             row[key] = normalize_path(
                 row[key], abbreviate_path_components=abbreviate_path_components
             )
-
     rows.sort(key=lambda r: r[key])
 
+    row_colors = color_match(rows)
     headers, rows = strip_empty_columns(rows)
 
     for r in rows:
         for x in r:
             if r.get(x) == 0:
                 r[x] = "-"
 
-    return totals, table(headers=[key] + headers, rows=rows, alignment=TABLE_ALIGNMENT)
-
+    return totals, table(headers=[key] + headers, rows=rows,row_colors = row_colors, alignment=TABLE_ALIGNMENT)
```

Comment:
Please run `black` over the code to make sure the formatting is consistent (https://pypi.org/project/black/)